app/src/layouts/adminLayout/adminSidebar/adminSidebar.jsx

```diff
@@ -142,13 +138,17 @@ export class AdminSidebar extends Component {
         message: <FormattedMessage id={'AdminSidebar.plugins'} defaultMessage={'Plugins'} />,
       },
     ];
-    const extensionItems = this.props.extensions.map((extension) => ({
-      onClick: this.handleClickButton(),
-      link: { type: PLUGIN_UI_EXTENSION_ADMIN_PAGE, payload: { pluginPage: extension.name } },
-      icon: extension.buttonIcon,
-      message: extension.buttonLabel || extension.name,
-    }));
-    return [...items, ...extensionItems];
+    if (extensions.length) {
```

Comment:
What is this check for?
forEach can operate on an empty array

Reply:
Fixed.